db/schema.rb

```diff
@@ -59,7 +60,7 @@
     t.string   "locale",                default: "en"
     t.string   "base_path"
     t.string   "title"
-    t.string   "description"
+    t.string   "old_description"
```

Comment:
I'm keeping the `old_description` fields for now (just in case). We can remove in another pull request.